ucte/ucte-converter/src/test/resources/sameId.uct

```diff
@@ -1,6 +1,6 @@
 ##C 2007.05.01
 Test case to check the import of a file with a line and a transformer with identical
-node-node-order "identifier".
+node-node-order "networkElementIdentifier".
```

Comment:
I think this is a mistake?

Reply:
yes probably a bad refactor from intellij